sql/indexes_with_boolean.sql

```diff
@@ -0,0 +1,15 @@
+select pi.indrelid::regclass::text   as table_name,
+       pi.indexrelid::regclass::text as index_name,
+       col.attname                   as column_name,
+       col.attnotnull                as column_not_null
+from
+    pg_catalog.pg_index pi
+    join pg_catalog.pg_class pc on pc.oid = pi.indexrelid
+    join pg_catalog.pg_namespace pn on pn.oid = pc.relnamespace
+    join pg_catalog.pg_attribute col on col.attrelid = pi.indrelid and col.attnum = any (pi.indkey)
+where pn.nspname = 'public'::text
```

Comment:
Use `:schema_name_param` here instead of `'public'`
https://github.com/mfvanek/pg-index-health-sql/blob/46fbf916c71a61239c6829e837057aa2e85b6180/sql/unused_indexes.sql#L28